MultiEditUtils.py

```diff
@@ -21,24 +48,86 @@ def run(self, edit, case=True, word=False, ignore_comments=False):
             newRegions.append(region)
 
       if word:
-        deleted = []
-        for region in newRegions:
-          if view.substr(view.word(region)).lower() not in selected_words:
-            deleted.append(region)
+        deleted = [region for region in newRegions
+                   if view.substr(view.word(region)).lower() not in selected_words]
         newRegions = [region for region in newRegions if region not in deleted]
 
       if ignore_comments:
-        deleted = []
-        for region in newRegions:
-          scope = view.scope_name(region.begin())
-          comment = re.search(r'\bcomment\b', scope)
-          if comment:
-            deleted.append(region)
-        newRegions = (region for region in newRegions if region not in deleted)
+        deleted = [region for region in newRegions
+                   if re.search(r'\bcomment\b', view.scope_name(region.a))]
+        newRegions = [region for region in newRegions if region not in deleted]
 
     for region in newRegions:
       view.sel().add(region)
 
+class MultiFindAllRegexCommand(sublime_plugin.TextCommand):
+

```

Comment:
This is a new command, additive or subtractive, it works on top of any selection/searches you made